pages/linux/ego.md

```diff
@@ -0,0 +1,32 @@
+# ego
+
+> Funtoo's official system personality management tool.
+> More information: <https://www.funtoo.org/Package:Ego>.
+
+- Synchronize the Portage tree:
+
+`ego sync`
+
+- Update bootloader configuration:
+
+`ego boot update`
+
+- Read a Funtoo wiki page:
+
+`ego doc {{wiki_page}}`
```

Comment:
I modify this example according to https://funtoo-ego.readthedocs.io/en/develop/ref/cli/ego-doc.1.html.
```suggestion
- Read the section from Funtoo wiki page by a given string:

`ego doc {{string}}`
```

Reply:
`wiki_page` is the way it's worded in the help message for `ego doc`. It's more accurate IMHO.
I'll make it a bit more descriptive though, I agree.